.github/workflows/main.yaml

```diff
@@ -256,7 +256,7 @@ jobs:
         with:
           entrypoint: bin/bloc.dart
```

Comment:
need to make this workflow more generic...right now the entrypoint is hard-coded